src/components/Bands.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import Band from './Band'
 
-const Bands = props => {
-    const bands = props.bands.map(band => <Band key={band.id} {...band} deleteBand={props.deleteBand} />)
-
-    return (
-        <div>
-            {bands}
-        </div>
-    )
-}
+const Bands = ({ bands, deleteBand }) => (
+  <ul className="bands-list">
+    {bands.length ? bands.map(band => (
+      <Band
+        key={band.id}
+        deleteBand={deleteBand}
+        {...band}
+      />
+    )) : (
+      <p className="empty-prompt-text">There are no bands, submit one (as long as it isn't Phish).</p>
+    )}
```

Comment:
Let's just map in place, rather than store a reference to data that we will then map.

Also, if `bands` doesn't have any data in it, lets prompt the user to give us some. Marketing will thank you.